tests/test_skyWcs.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import lsst.utils.tests
+from lsst.afw.geom import SkyWcs, SpherePoint, Extent2D, Point2D, degrees, arcseconds, makeCdMatrix
+from lsst.afw.geom.testUtils import TransformTestBaseClass
+
+
+class TanSkyWcsTestCase(TransformTestBaseClass):
+    """Test pure TAN constructor of SkyWcs
+    """
+
+    def setUp(self):
+        TransformTestBaseClass.setUp(self)
+        self.crpix = Point2D(100, 100)
+        self.crval = SpherePoint(30 * degrees, 45 * degrees)
+        self.scale = 1.0 * arcseconds
+
+    def checkTanWcs(self, orientation, flipX):
+        """Construct a pure TAN SkyWcs and check that it operates as specified
+
+        Parameters
+        ----------
+        orientation : lsst.afw.geom.Angle
+            Position angle of focal plane +Y, measured from N through E.
+            At 0 degrees, +Y is along N and +X is along E/W if flipX false/true
+            At 90 degrees, +Y is along E and +X is along S/N if flipX false/true
+        flipX : bool
+            Fip x axis? See `orientation` for details.
+
+        Returns
+        -------
+        wcs : SkyWcs
+            The generated pure TAN SkyWcs
+        """
+        wcs = SkyWcs(
+            crpix = self.crpix,
+            crval = self.crval,
+            cdMatrix = makeCdMatrix(scale=self.scale, orientation=orientation, flipX=flipX),
+        )
+        self.checkPersistence(wcs)
```

Comment:
I'm afraid I accidentally edited Paul's comment instead of my own. He said that Transform and SkyWcs should be pickleable. I agree and filed https://jira.lsstcorp.org/browse/DM-10416 to record that.